### ISSUE TITLE
Single cell image generation

### ISSUE THREAD

**Issue by MarziehHaghighi** [broadinstitute/2018_04_20_Rosetta#13]
## Generating CP images based on GE profiles:

![Screen Shot 2020-09-17 at 4 06 33 PM](https://user-images.githubusercontent.com/30931779/93522726-eaed7180-f8ff-11ea-859e-f07634def2e7.png)

* Note that the goal is very general but the dataset we have is very limited. In the above, the red line is the data at hand in this project:
  1 - Replicate/treatment levels l1k GE profiles
  2 - Single cell profiles/images of CP experiment

* 1 and 2 are two INDEPENDENT experiment and therefore samples are not paired in any ways.
* My personal opinion: Pairing methods are not suitable here. There are sutable for the case that we know that there is a pair for each cell/well but we don't know which one of is that one!

**Comment by MarziehHaghighi** [comment]
## Possible image generation scenarios:
* Map:  (treatment level GE profile) --> (blurred treatment level cell image)
   - Anne thinks it is useful, but I'm not convinced how it is more useful than just looking up important treatment level CP features and visualize it somehow (For example using the model I developed for visualizing cell state changes by tweeking each CP feature)

* Map:

**Comment by MarziehHaghighi** [comment]
## Evaluation metrics for the following model:
Map: (any single cell image) --> (single cell after gene expression )
Given a single cell image, and an specific target perturbation;
We want to visualize output of that perturbation and single cell GIVEN gene expression treatment level profile of that perturbation.   https://github.com/MarziehHaghighi/CellStateImageSynthesis/issues/5

**Comment by MarziehHaghighi** [comment]
### Updated qualitative+quantitative evaluation metric: 
#### Test set:
* We select 10 visually distinguishable perts for test (should have different GE profiles to training set profiles):
#### Metric:
* Biologist visual classification (into test reference 10 categories) results for real single cells versus synth cell images:


* Steps:
* For real images:
  1. We generate 10 random samples for each 10 pert class.
  2. There will be x rows that in each we have exactly one cell for each category.
  3. Biologist will classify each into one of the test reference classes. (examples are provided in a pdf file)

* For synth images:
  1. We randomly generate x DMSO cells and transform them into all the new test states. 
  2. Each row for evaluation, has DMSO cell as the first image and the shuffled synth generated test classes.
  3. Biologist will classify each into one of the test reference classes. (examples are provided in a pdf file)

**Comment by shntnu** [comment]
@MarziehHaghighi can reopen if needed
